src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static String getRandomBall() {
+        Ball ball = new Ball();
+        ColorSupplier color = new ColorSupplier();
```

Comment:
make `colorSupplier` and `random` class fields

Reply:
not fixed

Reply:
Each time the getRandomBall() method is called a ColorSupplier object is created, you don't need to do this, it only needs to be created once when you create the Lottery, so move it to the class level